### DISCUSSION TITLE
Selected text range through Accessibility APIs (macOS)

### DISCUSSION THREAD

**Discussion by joshmedeski** [wezterm/wezterm#5354] [Ideas]
Neovim currently has a problem with selected text in macOS. It doesn't correctly expose the selected text range through Accessibility APIs (it correctly exposes the cursor position, but not the selected text range). This forbids all the workflows depending on text selection (like Raycast).

@wez, have you done any work to make wezterm compatible with the selected text accessibility APIs on macOS?

**Comment by wez** [comment]
Not AFAIR. See also:
* https://github.com/wez/wezterm/issues/913
for a broader consideration of accessibility in the terminal
